Test/Tasty/Golden.hs

```diff
@@ -184,13 +205,21 @@ goldenVsFileDiff name cmdf ref new act =
   cmp sizeCutoff _ _
     | null cmd = error "goldenVsFileDiff: empty command line"
     | otherwise = do
-    -- On Windowss use_process_jobs indicates that we should wait for
-    -- the entire process tree to finish before unblocking.
-    -- On POSIX systems the flag is ignored.
     (_, Just sout, _, pid) <- createProcess
                                 (proc (head cmd) (tail cmd))
-                                { std_out = CreatePipe,
-                                  use_process_jobs = True }
+                                {  std_out = CreatePipe
+#ifdef MIN_VERSION_process
```

Comment:
Is this outer #ifdef necessary?

Reply:
It does seem cumbersome, but it is what the ghc manual shows https://downloads.haskell.org/ghc/latest/docs/html/users_guide/phases.html#standard-cpp-macros I assumed that it's there for some sort of compatibility reason, or to reduce compilation noise.

Reply:
Ah, but that's because that particular macro has been introduced recently, as they explain:

> This macro is available starting with GHC 7.10.1.
> ...
> If compatibility with Haskell compilers (including GHC prior to version 7.10.1) which do not define MIN_VERSION_GLASGOW_HASKELL is required, the presence of the MIN_VERSION_GLASGOW_HASKELL macro needs to be ensured before it is called, e.g.:

OTOH, MIN_VERSION_xxx macros for libraries are defined by Cabal and have been around since forever, so it's safe to assume they are defined.

Reply:
Thanks.  Zapped.